### DISCUSSION TITLE
Shared Translations Error

### DISCUSSION THREAD

**Discussion by genfish** [pimcore/pimcore#15345] [Q&A / Support] [answered]
I have a problem with the Shared Translations, the upper quote symbol ' is not interpreted correctly. It is necessary in the Catalan language.

Example text in English: Help Us
Example text in catalan: Ajuda'ns

Putting it there transforms into: Ajuda&amp;#039;ns

Thanks!

**Comment by genfish** [comment]
Not answered because hasn't solution? 🙄

**Reply by dvesh3** [reply]
Pimcore version please?

**Reply by dvesh3** [reply]
This is a bug on 11.0 version. I have created an issue https://github.com/pimcore/pimcore/issues/15438. Thanks for reporting 👍

**Reply by dvesh3** [reply, marked as answer]
Fixed by https://github.com/pimcore/admin-ui-classic-bundle/pull/183

**Comment by genfish** [comment]
Yes, work's in pimcore admin, but not in template page, when I use: {{ 'Participa y ayudanos'|trans }}

https://github.com/pimcore/pimcore/assets/9285058/88a55bf2-dad9-4ad2-933c-3ae1b0892d40

**Reply by dvesh3** [reply]
please see https://github.com/pimcore/pimcore/issues/15740
